lite_up/lib/constants/style.dart

```diff
@@ -0,0 +1,13 @@
+// This file contains customization for colors and fonts.
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+const TextStyle flashcardAppBarTitle =
+    TextStyle(color: Colors.black, fontSize: 24, fontWeight: FontWeight.bold);
+
+const Color background = Color.fromARGB(255, 249, 240, 181);
```

Comment:
Is background the soft yellow color? Perhaps naming it as yellow or softYellow would make it easier to find if we were to use it for non-backgrounds as it matches with the other naming conventions.